exercises/house/canonical-data.json

```diff
@@ -0,0 +1,322 @@
+          ]
+        },
+        {
+          "description": "verse three",
+          "property": "verse",
+          "verse": 3,
+          "expected": [
+            "This is the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse four",
+          "property": "verse",
+          "verse": 4,
+          "expected": [
+            "This is the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse five",
+          "property": "verse",
+          "verse": 5,
+          "expected": [
+            "This is the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse six",
+          "property": "verse",
+          "verse": 6,
+          "expected": [
+            "This is the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse seven",
+          "property": "verse",
+          "verse": 7,
+          "expected": [
+            "This is the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse eight",
+          "property": "verse",
+          "verse": 8,
+          "expected": [
+            "This is the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse nine",
+          "property": "verse",
+          "verse": 9,
+          "expected": [
+            "This is the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse 10",
```

Comment:
Just out of habit, I've learned to express numbers less than ten as words. Probably because of english class :stuck_out_tongue:. Do you think it would be better to stick with one or the other?